content/docs/06-integrations/00-aws-efs.md

```diff
@@ -0,0 +1,175 @@
+| provisioningMode | efs-ap           | efs-ap                | The type of volume provisioned by AWS EFS. For now, this is the <br /> only access point supported.                                                                                                                                      |
+| directoryPerms   |                  | 700                   | Directory permissions for Access Point root directory.creation.                                                                                                                                                                          |
+| gidRangeStart    |                  | 1000                  | Starting range of the Portable Operating System Interface(POSIX) group Id <br /> to be applied for access point root directory creation (optional).                                                                                      |
+| gidRangeEnd      |                  | 2000                  | End range of the POSIX group Id.(optional)                                                                                                                                                                                               |
+| basePath         |                  | /base_efs             | Path under which access points for dynamic provisioning is created. <br /> If this parameter is not specified, access points <br /> are created under the root directory of the file system.                                             |
+
+
+# Troubleshooting
+
+### Storage Class
+Storage classes created by Palette will be with the name *spectro-storage-class* and can be fetched from kubectl using the following CLI command:
+
+```bash
+kubectl get storageclass
+```
+
+### PersistentVolumeClaim
+
+The output of the kubectl describe pvc ${PVC_NAME} command is:
+
+ `$ PVC_NAME=efs; kubectl describe pvc ${PVC_NAME}`
+
+
+Name:          efs
+
+Namespace:     default
+
+StorageClass:  aws-efs
+
+Status:        Pending
+
+Volume:
+
+Labels:<none>
+
+Annotations:   kubectl.kubernetes.io/last-applied-configuration:
+{"apiVersion":"v1","kind":"PersistentVolumeClaim","metadata":{"annotations":{"volume.beta.kubernetes.io/
+storage-class":"aws-efs"},"name":"...
+
+volume.beta.kubernetes.io/storage-class: aws-efs
+
+Finalizers:    [kubernetes.io/pvc-protection]
+
+Capacity:
+
+Access Modes:
+
+Events:
+| Type    | Reason             | Age                | From                        | Message                  |
+| ------- | ------------------ | ------------------ | --------------------------- | ------------------------ |
+| Warning | ProvisioningFailed | 43s (x12 over 11m) | persistentvolume-controller | no volume plugin matched |
+Mounted By:  <none> </p>
+
+
+
+# References
+
+See the following information on all AWS EFS parameters:
```

Comment:
See the following information on all AWS EFS parameters: 
this sentence can be removed.